client/assets/javascripts/__tests__/globals/CharacterLimiter.test.js

```diff
@@ -0,0 +1,21 @@
+import { describe, it, expect, jest } from '@jest/globals'
+import CharacterLimiter from '../../globals/CharacterLimiter'
+
+describe('CharacterLimiter', () => {
+  it('should limit the number of characters on the required field', () => {
+    document.body.innerHTML = `
+    <div class="govuk-form">
+        <textarea class="govuk-textarea js-limit-chars-4" rows="5"></textarea>
+    </div>`
+
+    const textarea = document.querySelector('[class*="js-limit-chars-"]')
```

Comment:
Is this JS just to avoid using the maxlength attribute on fields?

Reply:
Believe so -- which now makes me think we don't actually need it and I can delete it :)